### DISCUSSION TITLE
Export only posts and pages

### DISCUSSION THREAD

**Discussion by glaslos** [honeynet/honeynet.github.io#12] [General]
I've been messing around a bit in the current code based and I have the feeling the current approach is very messy. We are trying to replicate the old site but by doing so we bring a lot of clutter in the form of templates and scripts. I'd suggest to only export posts and pages and start a page setup from scratch (includes and layouts).

**Comment by aikinci** [comment]
I totally agree. If we can we should port the site to a Markdown only version if not discard or archive the old site and start over anew. This legacy is only holding us back!

**Comment by hgascon** [comment]
I used recently the minimalmistakes theme for jekyll to create a static site from scratch and it was very easy to adapt. Maybe we can check that. Also, @glaslos in the other branch that is not "development", I used another tool to export the wordpress xml to jekyll. I think the output was a bit more structured. You might want to check it. Let me know if you have questions there.

**Reply by aikinci** [reply]
We have it here https://github.com/honeynet/www-static/blob/main/thehoneynetproject.WordPress.2022-12-05.xml 
please try and see if you can get it working.

**Reply by xandfury** [reply]
@aikinci I think @hgascon meant this branch https://github.com/honeynet/honeynet.github.io/tree/xml-to-markdown
It already is doing the xml to jekyll. 😄

**Comment by xandfury** [comment]
Not sure why I stopped receiving notifications about this. Strange 😕 Anyway. I have no issues with this. In fact, I had briefly discussed with @dpisano about just keeping the markdown files. My main problem was with the licences of "betheme (current theme on wordpress)" which might not be compatible with our open source website ambitions 😄 

I have been trying to build something with [`chirpy`](https://github.com/cotes2020/jekyll-theme-chirpy) theme. It's under MIT license. I haven't updated it on the repo yet since I wanted to discuss it with everyone first 😄

**Reply by xandfury** [reply]
Here are some screen shots. It's kinda modified.

### 1 
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/4391664/232750720-6ee7d753-e48b-4991-8f16-56dc4c76d979.png">

### 2
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/4391664/232750933-b46bc1f1-e9a2-4a6c-9bdc-ef817ed6fe3c.png">

**Reply by xandfury** [reply]
I really like Tor's website. ([link](https://www.torproject.org/)). Especially the part where they have included the donate button as part of the logo 😉 So I wanted to build something like that.. 

The topbar you see in pictures can be something like this.
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/4391664/232751607-32d8a1ab-da87-434a-b82f-094c30a988e0.png">

Happy to bounce off ideas on a call. 😄

**Reply by xandfury** [reply]
I have the code for the updates in a separate private repo. But can update it here. If I get everyone's blessing to do so.

**Reply by hgascon** [reply]
Maybe test it in a new branch?

**Reply by xandfury** [reply]
@hgascon updated in [`theme`](https://github.com/honeynet/honeynet.github.io/tree/theme) branch.
- fixed several pages as well
- init internationalization (different languages)
cc: @glaslos @aikinci
